packaging/nnstreamer.spec

```diff
@@ -97,6 +99,14 @@ make %{?_smp_mflags}
 ./tests/unittest_sink --gst-plugin-path=.
 ./tests/unittest_plugins --gst-plugin-path=.
 popd
+
```

Comment:
Please try to the aging test if this PR can be effective before doing the merge process.